### PR TITLE
Add setting to support different file extension

### LINKED ISSUE #12: Ability to use .js instead of jsx
Good job on the extension! 
Curious to know if there a way for us to use only JS instead of JSX ? 
PR's are welcomed, if so Il create one for it

### PR DESCRIPTION
**Pull request by etiennejcharles**
Would like to be able to us "ts" instead of "tsx" or "js" instead of "jsx"
Fixes: https://github.com/yeet-bix/create-react-component-vscode/issues/12
Create a PR for it 😉

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -88,6 +88,15 @@
...
                 },
+                "createReactComponent.fileExtension": {
+                    "type": "string",
+                    "enum": [
+                        "withX",
+                        "withoutX"
+                    ],
+                    "default": "withX",
+                    "description": "Specifies the extension type of files: .ie 'withX' would create 'MyComponent.tsx' or 'MyComponent.jsx'. 'withoutX' would create 'MyComponent.ts' or 'MyComponent.js' files"
+                },
```
Comment (yeet-bix) [inline package.json:99]: Could you please add some documentation for this setting in the README? 😄 
https://github.com/yeet-bix/create-react-component-vscode/#settings

File: src/template/templateOptions.ts [hunk 2]
```diff
@@ -14,11 +14,17 @@ export enum FileType {
     Index = "index"
 }
 
+export enum FileExtension {
+    Tsx = "withX",
+    Ts = "withoutX",
+}
```
Comment (yeet-bix) [inline src/template/templateOptions.ts]:
```suggestion
export enum FileExtensionType {
    WithX = "withX",
    WithoutX = "withoutX",
}
```
Small nitpick: `FileExtension` implies to me it's the values of file extensions, e.g `FileExtension.Tsx` = `.tsx`

**Review by yeet-bix — COMMENTED** [review]
Awesome stuff! Just a small change and docs and we can get this merged and released ASAP 🚀

File: package.json (hunk 1 above)
Comment (etiennejcharles) [reply package.json:99]: Done!

File: src/template/templateOptions.ts (hunk 2 above)
Comment (etiennejcharles) [reply src/template/templateOptions.ts]: Done!

**Review by yeet-bix — APPROVED** [review]
@etiennejcharles thanks for the contribution! :shipit:

**Comment by etiennejcharles** [comment]
Sweet! 🥳

**Pull request merged**